documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/cadl-project.yaml

```diff
@@ -21,7 +21,7 @@ options:
     "emitter-output-dir": "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
     "package-mode": "dataplane"
   "@azure-tools/cadl-csharp":
-    namespace: Azure.Contoso.Widgetmanager
+    namespace: Azure.Template.Contoso
```

Comment:
Why only change namespace for csharp? how about java?  

Another inconsistent thing is the `emitter-output-dir` of typescript, can it be changed to use value of namespace?

Reply:
namespace `Azure.Contoso` is not registered and .Net sdk pipeline reports error. So we need to reuse an existing test namespace.